airbyte-connector-builder-server/build.gradle

```diff
@@ -13,9 +13,9 @@ airbytePython {
 task generateOpenApiPythonServer(type: GenerateTask){
     outputs.upToDateWhen { false }
 
-    def generatedCodeDir = "$buildDir/server"
+    def generatedCodeDir = "$buildDir/airbyte_connector_builder_server"
```

Comment:
since the module name change, we weren't moving the models from `/build` to `/generated`. This should address that